app/code/extensions/ExtraConfig.php

```diff
@@ -1,19 +1,19 @@
 <?php
 
-namespace App\extensions;
+namespace App\Extensions;
 
 use SilverStripe\Assets\Image;
 use SilverStripe\Forms\FieldList;
 use SilverStripe\Forms\TextField;
 use SilverStripe\ORM\DataExtension;
-use SilverStripe\Forms\NumericField;
+use SilverStripe\CMS\Model\SiteTree;
+use SilverStripe\Forms\TreeDropdownField;
 use SilverStripe\AssetAdmin\Forms\UploadField;
-use SilverStripe\Forms\HTMLEditor\HTMLEditorField;
 
-class FooterConfig extends DataExtension
+class ExtraConfig extends DataExtension
 {
-
```

Comment:
Possibly add a table name to get rid of that error that comes up when you `dev/build`

Reply:
No need to add table name if it is within Settings page.